README.md

```diff
@@ -30,16 +30,53 @@ month: 2
 year: 2023
 ---
 
-your markdown content
+# This is a heading style one
+
+**This is a featured Paragraph it cannot contain bold text but can contain [links](https://www.unitary.fund).**
+
+More text
+
+::image[d394zg4utwqxynenamqf]
+
+## This is a heading in heading style two
+
+This is a standard paragraph with a [LINK](https://www.google.com). **This is bold paragraph text**
+
+- this is a list item
+- this is **bold** list item
+- this is a long list item which has lots of words in it and will probably break onto a second or even a third line so we can see how that looks at differnt sizes adn make sure it looks as it is intended too
+- this is a list item with a [LINK](https://www.google.com) or [two)](https://www.google.com) added in it. 
+
+### This is a heading in heading style three introducing a video
+
+https://www.youtube.com/watch?v=dB_3R84ewig
+
+#### This is a heading in heading style four
+
+
+## Including code snippets with gists
+A supported way to include code snippets is by embedding gists. Gists are code snippets that you can create on Github. Make sure that they are public. For example, navigating to [gist.github.com/matt-lourens/](https://gist.github.com/matt-lourens/) one can click on one of the gists and obtain the url  https://gist.github.com/matt-lourens/6cc14d37209de07abd707804f1b0219e. In order to embed the related code in the blog post, you can do one of the following:
+
+::gist[6cc14d37209de07abd707804f1b0219e]
+
+or with slashes
+
+::gist[matt-lourens/6cc14d37209de07abd707804f1b0219e/]
 ```
 
 Once your post is ready, open a pull request (PR)!
+
 The Vercel automation will create a preview you can access by clicking the "Visit preview" link that shows up in the bot comment.
 If the changes look good, request a review by tagging one of the UF team members (e.g., `@nathanshammah`).
 
-### Adding images to a blog post
+## Adding images
+
+To add an image in a page, there are two options:
+
+1. (**No Code option**): Go to https://unitary-fund.vercel.app/author/image and upload your desired image. 
+ Copy the code it returns after upload and use such and such syntax to insert. (e.g, d394zg4utwqxynenamqf). Note that this option is the required one for adding a cover image to a new `event`.
```

Comment:
I think you may have copy and pasted my suggestion from https://github.com/unitaryfund/unitary.fund/pull/359#discussion_r1316369594 which was a template, but did not include all the details (which is why I did not use the suggestion tool). E.g. "such and such syntax" needs replaced.